lib/web_ui/test/canvaskit/common.dart

```diff
@@ -113,33 +115,31 @@ class TestCollector implements Collector {
       }
       if (activeRegistration == null) {
         late final _TestFinalizerRegistration? collectedRegistration;
-        for (_TestFinalizerRegistration registration in _collectedRegistrations) {
+        for (_TestFinalizerRegistration registration
+            in _collectedRegistrations) {
           if (identical(registration.deletable, collection.deletable)) {
             collectedRegistration = registration;
             break;
           }
         }
         if (collectedRegistration == null) {
           fail(
-            'Attempted to collect an object that was never registered for finalization.\n'
-            'The collection was requested here:\n'
-            '${collection.stackTrace}'
-          );
+              'Attempted to collect an object that was never registered for finalization.\n'
+              'The collection was requested here:\n'
+              '${collection.stackTrace}');
         } else {
-          final _TestCollection firstCollection = _completedCollections.firstWhere(
-            (_TestCollection completedCollection) {
-              return identical(completedCollection.deletable, collection.deletable);
-            }
-          );
-          fail(
-            'Attempted to collect an object that was previously collected.\n'
-            'The object was registered for finalization here:\n'
-            '${collection.stackTrace}\n\n'
-            'The first collection was requested here:\n'
-            '${firstCollection.stackTrace}\n\n'
-            'The second collection was requested here:\n'
-            '${collection.stackTrace}'
-          );
+          final _TestCollection firstCollection = _completedCollections
+              .firstWhere((_TestCollection completedCollection) {
+            return identical(
+                completedCollection.deletable, collection.deletable);
+          });
+          fail('Attempted to collect an object that was previously collected.\n'
+              'The object was registered for finalization here:\n'
+              '${collection.stackTrace}\n\n'
+              'The first collection was requested here:\n'
+              '${firstCollection.stackTrace}\n\n'
+              'The second collection was requested here:\n'
+              '${collection.stackTrace}');
```

Comment:
I think dartfmt might do a better job if there was a trailing comma after the message.

Reply:
Done.